polars/polars-lazy/polars-plan/Cargo.toml

```diff
@@ -10,6 +10,7 @@ description = "Lazy query engine for the Polars DataFrame library"
 
 [dependencies]
 ahash.workspace = true
+chrono-tz = "0.8.1"
```

Comment:
This should be an optional dependency and only activate when the feature `"timezones"` is activated.

See the `Cargo.toml` in `polars-time` for an example of this.